recipes/ibis-framework/bld.bat

```diff
@@ -0,0 +1,10 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
+
+"%PYTHON%" -c "import ibis; print(ibis.__version__)" > __conda_version__.txt
```

Comment:
I pulled it from the recipe that ships with ibis.  was there for making nicely versioned files from master builds.  conda will read the version and num and string from those files if supplied